docs/en/04_Changelogs/4.0.0.md

```diff
@@ -829,39 +829,16 @@ guide for configuration instruction.
 
 #### Migrating File DataObject from 3.x to 4.0
 
-Since the structure of `File` dataobjects has changed, a new task `MigrateFileTask` has been added to assist
-in migration of legacy files. Migration can be invoked by either this task, or can be configured to automatically
-run during dev build by setting the `File.migrate_legacy_file` config to true. However, it's recommended that
-this task is run manually during an explicit migration process, as this process could potentially consume
-large amounts of memory and run for an extended time.
+Since the structure of `File` dataobjects has changed between 3.0 and 4.0, a new task `MigrateFileTask`
+has been added to assist in migration of legacy files.
 
+You can run this task on the command line:
 
-```yml
-SilverStripe\Assets\File:
-  migrate_legacy_file: true
 ```
-
-This task will also support migration of existing File DataObjects to file versioning. Any
-pre-existing File DataObjects will be automatically published to the live stage, to ensure
-that previously visible assets remain visible to the public site.
-
-If additional security or visibility rules should be applied to File dataobjects, then
-make sure to correctly extend `canView` via extensions.
-
-Note that any File dataobject which is not in the `File.allowed_extensions` config will be deleted
-from the database during migration. Any invalid file on the filesystem will not be deleted,
-but will no longer be attached to a dataobject anymore, and should be cleaned up manually.
```

Comment:
I would strongly encourage this note in the change logs, as it's quite important and easily missed if not here.